README.md

```diff
@@ -737,7 +737,7 @@ This makes it very easy to achieve powerful patterns similar to sideways data lo
 
 `createTransformer` turns a function (that should transform value `A` into another value `B`) into a reactive and memoizing function.
 In other words, if the `transformation` function computes B given a specific A, the same B will be returned for all other future invocations of the transformation with the same A.
-However, if A changes, the transformation will be re-applied so that B is updated accordingly.
+However, if A changes, or any derivation accessed in the transformer function body gets invalidated, the transformation will be re-applied so that B is updated accordingly.
```

Comment:
```suggestion
However, if `A` changes, or any derivation accessed in the transformer function body gets invalidated, the transformation will be re-applied so that `B` is updated accordingly.
```